eZ/Publish/Core/Persistence/Cache/Readme.md

```diff
@@ -0,0 +1,72 @@
+# CORE DEV DOC: Persistence\Cache
```

Comment:
Probably the whole document needs to be reviewed by the @ezsystems/documentation-team.

Reply:
The real question is: do we really want to have this here instead of in the doc? The PR description says `Doc needed | no`, so I'm not sure what's the role of this file.

Reply:
This is doc for engineering and contributors who needs to know how things work internally and avoid repeating mistakes in the future, not for partners.

_So this is not for doc.ezplatform.com at all, but I think @alongosz suggested it to be doc team approved anyway._

Reply:
Maybe it should be included in internal developer guidelines / code standards, once they are codified?
(Anyway, will give the file a quick look)

Reply:
Code / Core doc should really be close to the code imo, but otherwise 👍 It's basically doc for maintainers & contributors specifically, internal and external.